src/main/java/org/broadinstitute/hellbender/tools/reference/CompareReferences.java

```diff
@@ -204,6 +256,193 @@ private void displayBySequenceName(Set<ReferenceSequenceTable.TableRow> rows, St
+                    GATKPath ref2Fasta = generateFastaForSequence(ReferenceDataSource.of(ref2Path.toPath(), true), sequenceName, new GATKPath(ref2TempFastaOutput.toString()));
+
+                    // pass fastas into mummer, get back a vcf and add to list
+                    MummerExecutor executor = new MummerExecutor();
+                    logger.info("Running mummer alignment on sequence " + sequenceName);
+                    File tempSnpsDirectory = IOUtils.createTempDir("tempsnps");
+                    File mummerOutput = executor.executeMummer(ref1Fasta.toPath().toFile(), ref2Fasta.toPath().toFile(), tempSnpsDirectory);
+                    logger.info("Finished running mummer alignment on sequence " + sequenceName);
+                    snpsFiles.add(mummerOutput);
+                }
+            }
+            // merge individual snps files
+            File snps = IOUtils.createTempFile(String.format("%s_%s", refPair.getRef1AsString(), refPair.getRef2AsString()), ".snps");
+            /*new File(baseComparisonOutputDirectory.toPath().toString(), String.format("%s_%s.snps", refPair.getRef1AsString(), refPair.getRef2AsString()));*/
+            try (PrintWriter writer = new PrintWriter(snps)) {
+                for (File file : snpsFiles) {
+                    try (XReadLines reader = new XReadLines(file)) {
+                            for (String line : reader) {
+                                writer.write(line + "\n");
+                            }
+                    } catch (IOException e) {
+                        throw new UserException("Error merging show-snps outputs.", e);
+                    }
+                }
+            }
+            catch(IOException e){
+                throw new UserException("Error writing show-snps output file.");
+            }
+            File vcf = new File(baseComparisonOutputDirectory.toPath().toString(), String.format("%s_%s.vcf", refPair.getRef1AsString(), refPair.getRef2AsString()));
+            try(XReadLines reader = new XReadLines(snps);
+                VariantContextWriter writer = createVCFWriter(vcf);
+                ReferenceDataSource source = ReferenceDataSource.of(refPair.getRef1().toPath())){
+                VCFHeader header = new VCFHeader();
+                header.setSequenceDictionary(getReferenceDictionary());
+                writer.writeHeader(header);
+                MummerIndel currentIndel = null;
+                int previousPos = -1;
+                    for (String line : reader) {
+                        String[] fields = line.split("\\t", -1);
+                        String contig = fields[12];
+                        int pos = Integer.valueOf(fields[0]);
+                        String ref = fields[1];
+                        String alt = fields[2];
+
+                        // insertion
+                        if (ref.equals(".") && !alt.equals(".")) {
+                            if (pos == previousPos && currentIndel.isInsertion) {
```

Comment:
Need to make sure currentIndel != null here as well:

`if (pos == previousPos && currentIndel != null && currentIndel.isInsertion)`